diff --git a/src/gl/glwindow.cpp b/src/gl/glwindow.cpp
index c8506ac..8165a8e 100644
--- a/src/gl/glwindow.cpp
+++ b/src/gl/glwindow.cpp
@@ -47,7 +47,7 @@ void GLWindow::create(const char* title, int width, int height)
 // 	SDL_WM_SetIcon(SDL_LoadBMP("other files/critterding icon.png"), 0);
 // 	SDL_WM_SetIcon(SDL_LoadBMP("other files/cd.bmp"), 0);
 	string pixmappath = Settings::Instance()->binarypath;
-	pixmappath.append( "../share/critterding/pixmaps/cd.bmp" );
+	pixmappath.append( "@FINKPREFIX@/share/critterding/pixmaps/cd.bmp" );
 	SDL_WM_SetIcon(SDL_LoadBMP(pixmappath.c_str()), 0);
 // 	SDL_WM_SetIcon(SDL_LoadBMP("/projects/lessons/lesson20/data/image2.bmp"), 0);
 
diff --git a/src/gui/textprinter.cpp b/src/gui/textprinter.cpp
index eec8e7c..f631492 100644
--- a/src/gui/textprinter.cpp
+++ b/src/gui/textprinter.cpp
@@ -185,8 +185,8 @@ void Textprinter::setUpFonts()
 	char const *fontFilePath;
 
 	string fontpath = Settings::Instance()->binarypath;
-	string defaultf = "../share/critterding/font.ttf";
-	string systemf = "/usr/share/fonts/TTF/DejaVuSans.ttf";
+	string defaultf = "@FINKPREFIX@/share/critterding/font.ttf";
+	string systemf = "/Library/Fonts/Verdana.ttf";
 
 	string usepath = fontpath;
 	usepath.append(defaultf);
